frontend/src/components/Pages/Dashboard/UploadVideo.jsx

```
@@ -1,559 +1,847 @@
+      setIsAutoCropping(false);
+      reject(new Error("Canvas reference not available"));
+      return;
+    }
+
+    const ctx = canvas.getContext('2d');
+    const { width: sourceWidth, height: sourceHeight } = img;
+    
+    canvas.width = TARGET_WIDTH;
+    canvas.height = TARGET_HEIGHT;
+    
+    let drawWidth, drawHeight, offsetX = 0, offsetY = 0;
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)